sql/base/LegionCore_auth_2020_04_03.sql

```diff
@@ -553,7 +555,7 @@ CREATE TABLE `store_category_locales` (
   `description_it` varchar(255) NOT NULL DEFAULT '',
   `description_ua` varchar(255) NOT NULL DEFAULT '',
   PRIMARY KEY (`category`) USING BTREE
-) ENGINE=InnoDB DEFAULT CHARSET=utf8 ROW_FORMAT=COMPACT;
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 ROW_FORMAT=COMPRESSED;
```

Comment:
Is this required or a nice to have?

Reply:
This was required to get that table to create in MySQL 8 due to the row size being too large error I received.  Could be worth a look to see what other alternatives could be done.

Reply:
Do you still have the specific error available so I can take a closer look?
Because if I look around a bit it seems the `COMPRESSED` row format is not amazing.
https://jira.mariadb.org/browse/MDEV-23497

Reply:
Yeah I didn't like the compressed, I believe the alternative is changing the page size in MySQL 8 to accommodate this super sized table row.   I believe this was the error:

[Error Code: 1118 Row size too large (> 8126). Changing some columns to TEXT or BLOB](https://stackoverflow.com/questions/22637733/mysql-error-code-1118-row-size-too-large-8126-changing-some-columns-to-te)

Reply:
Seems to be introduced in MySQL 5.7: https://stackoverflow.com/questions/22637733/mysql-error-code-1118-row-size-too-large-8126-changing-some-columns-to-te
With more info here:
https://mariadb.com/kb/en/troubleshooting-row-size-too-large-errors-with-innodb/
Might be worth checking which table(s) is problematic, if only a couple I would prefer fixing the root cause then.
Not promising I will check this, but with the instructions on the last link, it seems quite feasible to quickly go through all the tables.